lock.rst

```diff
@@ -129,32 +129,34 @@ this behavior by using the ``lock`` key like:
     .. code-block:: php
 
         // config/packages/lock.php
-        $container->loadFromExtension('framework', [
-            'lock' => null,
-            'lock' => 'flock',
-            'lock' => 'flock:///path/to/file',
-            'lock' => 'semaphore',
-            'lock' => 'memcached://m1.docker',
-            'lock' => ['memcached://m1.docker', 'memcached://m2.docker'],
-            'lock' => 'redis://r1.docker',
-            'lock' => ['redis://r1.docker', 'redis://r2.docker'],
-            'lock' => 'zookeeper://z1.docker',
-            'lock' => 'zookeeper://z1.docker,z2.docker',
-            'lock' => 'sqlite:///%kernel.project_dir%/var/lock.db',
-            'lock' => 'mysql:host=127.0.0.1;dbname=app',
-            'lock' => 'pgsql:host=127.0.0.1;dbname=app',
-            'lock' => 'pgsql+advisory:host=127.0.0.1;dbname=lock',
-            'lock' => 'sqlsrv:server=127.0.0.1;Database=app',
-            'lock' => 'oci:host=127.0.0.1;dbname=app',
-            'lock' => 'mongodb://127.0.0.1/app?collection=lock',
-            'lock' => '%env(LOCK_DSN)%',
-
-            // named locks
-            'lock' => [
-                'invoice' => ['semaphore', 'redis://r2.docker'],
-                'report' => 'semaphore',
-            ],
-        ]);
+        use Symfony\Config\FrameworkConfig;
+
+        return static function (FrameworkConfig $framework) {
+            $framework->lock()
+                ->resource('lock', null)
```

Comment:
This should be `'default'` which is the internal value used when the user did not provide named resource

Reply:
Is this what you mean?

```suggestion
                ->resource('lock', ['default'])
```

Reply:
```suggestion
                ->resource('default', null)
                ->resource('default', ['flock'])
```

In the previous version, the word `'lock'` was for `framework.lock`, which is now replaced by `$framework->lock()`

Reply:
hmm I'm not sure that `null` is allowed with this new syntax

Reply:
Technically, no =)

It violates the contract by PHP doc block. But there is no complaint as long as the Config Definition supports it.